utility/src/main/java/org/oppia/util/data/UrlImageParser.kt

```diff
@@ -0,0 +1,71 @@
+package org.oppia.util.data
```

Comment:
Overall, I have a few nits about improving this class once it's in its own PR, but I think the approach is reasonable. I wish we didn't need to do manual drawing logic, but it seems unavoidable with how we're using Glide. Please add a TODO on #169 to simplify this solution once we can just fetch bitmaps for a filename since they'll be predownloaded with the exploration. I think we can actually implement a synchronous ImageGetter at that point which would be great.

Reply:
added a Todo with exploration asset downloader